packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -67,6 +67,7 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
   void paint(PaintingContext context, Offset offset) {
     context.canvas.save();
     context.canvas.translate(offset.dx, offset.dy);
+    context.canvas.clipRect(Rect.fromLTWH(0, 0, size.width, size.height));
```

Comment:
You don't want to create a new `Rect` here on every tick.

Reply:
ok